internal/featuresupport/support_levels_list.go

```diff
@@ -262,12 +286,16 @@ var SupportLevelsList = models.FeatureSupportLevels{
 				FeatureID:    models.FeatureSupportLevelFeaturesItems0FeatureIDVIPAUTOALLOC,
 				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelDevPreview,
 			},
+			{
+				FeatureID:    models.FeatureSupportLevelFeaturesItems0FeatureIDNUTANIXINTEGRATION,
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelUnsupported,
```

Comment:
why it's unsupported in 4.12? it should be FeatureSupportLevelFeaturesItems0SupportLevelSupported

Reply:
Right, moved this to "Supported"

Reply:
Fixed, moved to supported